install-service-runner-update.md

```diff
@@ -14,16 +14,16 @@ The process is as follows:
 
 ```
 sudo pip install redis
-sudo ln -s /home/pi/emonpi/service-runner.service /etc/systemd/system
+sudo ln -s /home/pi/emonpi/service-runner.service /lib/systemd/system
```

Comment:
Or we just simply have alternate install instructions for Jessie / Stretch.

Reply:
Mm it seams I get the same `No such file or directory` when I try and enable the service-runner service on Jessie using `/etc/systemd/system` even after `daemon-reload`. Any ideas? No that this is a big issue, I'm happy just to entertain this updated service runner for Stretch only

Reply:
> However since /etc/systemd/system works on both Jessie and Stretch, could I propose we use this location? Are there any disadvantages to this?

The disadvantage is described at the top of my comment in the other PR:
https://github.com/openenergymonitor/emonpi/pull/66#issuecomment-416114841

I think I have a jessie install here, I'll see what `pkg-config systemd --variable=systemdsystemunitdir` says there...

Reply:
Ok, on my jessie install I get this:
```
pi@emonpi(ro):~$ pkg-config systemd --variable=systemdsystemunitdir
/lib/systemd/system
```
Which *should* mean (according to the man page) that is the right place for unit files to go.

It is certainly full of systemd unit files on my jessie emonPi SD card.

Reply:
This is from the `man systemd` page on jessie:
>DIRECTORIES\
   System unit directories\
The systemd system manager reads unit configuration from various directories. Packages that want to install unit files shall place them in the directory returned by pkg-config systemd --variable=systemdsystemunitdir. Other directories checked are /usr/local/lib/systemd/system and
/lib/systemd/system. User configuration always takes precedence.  pkg-config systemd --variable=systemdsystemconfdir returns the path of the system configuration directory. Packages should alter the content of these directories only with the enable and disable commands of the systemctl(1) tool. Full list of directories is provided in systemd.unit(5).

this is from the `man systemd` page on stretch:
>DIRECTORIES\
       System unit directories\
           The systemd system manager reads unit configuration from various directories. Packages that want to install unit files shall place them in the directory returned by pkg-config systemd --variable=systemdsystemunitdir. Other directories checked are /usr/local/lib/systemd/system and /lib/systemd/system. User configuration always takes precedence.  pkg-config systemd --variable=systemdsystemconfdir returns the path of the system configuration directory. Packages should alter the content of these directories only with the enable and disable commands of the systemctl(1) tool. Full list of directories is provided in systemd.unit(5).

Both of those say the same thing...
>Packages that want to install unit files shall place them in the directory returned by pkg-config systemd --variable=systemdsystemunitdir

Which on both my stretch and jessie installs return `/lib/systemd/system` for me.
This:
```
pi@emonpi(rw):emonpi$ sudo systemctl enable service-runner.service
Failed to execute operation: No such file or directory
```
feels like a bug in the jessie systemd.

As for your other comment:
>Or we just simply have alternate install instructions for Jessie / Stretch.

I think we go with this :)